src/components/pages/PendingApplications/PendingApplication.js

```diff
@@ -4,15 +4,30 @@ import ApplicationModal from './ApplicationModal';
 import { Table, Button, Tag } from 'antd';
 import './PendingApplication.css';
 
+// Filter by status
+const statusFilter = (value, record) => {
+  if (Array.isArray(value)) {
+    return (
+      record.status.props.children === value[0] ||
+      record.status.props.children === value[1] ||
+      record.status.props.children === value[2]
+    );
+  } else {
+    return record.status.props.children === value;
+  }
+};
+
 const columns = [
+  // Names sorting by alphabetical order
   {
     title: 'Name',
     dataIndex: 'name',
     key: 'name',
-    defaultSortOrder: 'descend',
```

Comment:
I **REALLY** like the way the sort ability works in this component. Great use of the AntD components!